docs/aws/README.md

```diff
@@ -11,36 +13,61 @@ export AWS_DEFAULT_OUTPUT=json
 ```
 
 ### Create a Cluster
-Note: If you already have a cluster with version 1.19 or above, you may need to manually tag your subnets for Karpenter to work [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
 
-If your cluster version is 1.18 or below, you can skip this step.
+Create an EKS cluster
 ```bash
 eksctl create cluster \
 --name ${CLUSTER_NAME} \
---version 1.18 \
---region ${AWS_DEFAULT_REGION} \
 --node-type m5.large \
 --nodes 1 \
 --nodes-min 1 \
 --nodes-max 10 \
---managed
+--managed \
+--with-oidc
```

Comment:
Do you know if there's a difference between using this flag in `eksctl create cluster` vs `eksctl utils associate-iam-oidc-provider` ?

Reply:
If --with-oidc replaces it, that would be `sweet`

Reply:
`--with-oidc` does the same thing. One can just be performed post-cluster creation.